apps/src/code-studio/pd/workshop_dashboard/EnrollmentsPanel.jsx

```diff
@@ -0,0 +1,230 @@
+import $ from 'jquery';
+import React from 'react';
+import PropTypes from 'prop-types';
+import {Button} from 'react-bootstrap';
+import moment from 'moment';
+import MoveEnrollmentsDialog from './components/move_enrollments_dialog';
+import Spinner from '../components/spinner';
+import WorkshopEnrollment from './components/workshop_enrollment';
+import WorkshopPanel from './WorkshopPanel';
+
+/**
+ * View and manage the list of teachers enrolled in a workshop.
+ */
+export default class EnrollmentsPanel extends React.Component {
```

Comment:
Note: I'm especially excited about extracting `EnrollmentsPanel` because this is where we'll give admins the ability to change enrollment details in the near future.  There's definitely some further simplification that could happen in this component but I picked this as a stopping point since we'll be coming back to it soon anyway.